plugin/trino-hive/src/main/java/io/trino/plugin/hive/HiveConfig.java

```diff
@@ -475,15 +475,15 @@ public HiveConfig setHiveStorageFormat(HiveStorageFormat hiveStorageFormat)
         return this;
     }
 
-    public HiveCompressionCodec getHiveCompressionCodec()
+    public HiveCompressionOption getHiveCompressionOption()
     {
-        return hiveCompressionCodec;
+        return hiveCompressionOption;
     }
 
     @Config("hive.compression-codec")
-    public HiveConfig setHiveCompressionCodec(HiveCompressionCodec hiveCompressionCodec)
+    public HiveConfig setHiveCompressionOption(HiveCompressionOption hiveCompressionOption)
```

Comment:
you can keep old property name, since config name not changed (they should match and they shouldn't be changed at this point)

Reply:
changed back (all the way down to session properties)